public/locales/en-US/translation.json

```diff
@@ -0,0 +1,20 @@
+{
+  "common": {
+    "cancelButton": "Cancel"
+  },
+  "modal": {
+    "delete": {
```

Comment:
modal.delete is not very descriptive. What are we deleting? Currently we have only strategies but lets prepare for other possible features in the future where we would have another delete modal.